### PR TITLE
UI clean up

### PR DESCRIPTION
**Pull request by tomhebunn**
Made a few styling updates across the UI to improve the overall look and feel of the platform. Main addition - footer component

### CODE REVIEW

File: src/components/viewAssessment/ViewAssessmentButtons.vue [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+    <!-- <v-btn x-large icon color="primary">
       <v-icon>mdi-cog</v-icon>
     </v-btn>
     <v-btn x-large icon color="primary">
       <v-icon>mdi-share-variant</v-icon>
     </v-btn>
     <v-btn x-large icon color="primary">
       <v-icon>mdi-open-in-new</v-icon>
-    </v-btn>
+    </v-btn> -->
```
Comment (Exilliar) [inline src/components/viewAssessment/ViewAssessmentButtons.vue:11]: if the commented html isn't needed anymore it can be deleted

**Review by Exilliar — COMMENTED** [review]
Looking good! Got one small comment below, then should be good to merge

File: src/components/viewAssessment/ViewAssessmentButtons.vue (hunk 1 above)
Comment (tomhebunn) [reply src/components/viewAssessment/ViewAssessmentButtons.vue]: resolved

**Review by Exilliar — APPROVED** [review]
looks good, can merge once the checks pass (they're currently running while I'm writing this)

**Pull request merged**